certified-connectors/Alkymi/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1301 @@
+{
```

Comment:
@tori-alkymi If it is an independent publisher connector, the connector artifacts should be placed under the "independent-publisher connectors" folder.

Reply:
@tori-alkymi Can you please let us know why this comment was resolved?

Reply:
Because this is a submission for a certified connector.